lib/db/set.go

```diff
@@ -285,21 +371,47 @@ func (s *FileSet) ListDevices() []protocol.DeviceID {
 // database.
 func DropFolder(ll *Lowlevel, folder string) {
 	db := newInstance(ll)
-	db.dropFolder([]byte(folder))
-	db.dropMtimes([]byte(folder))
-	db.dropFolderMeta([]byte(folder))
```

Comment:
```
for f := range []func([]byte) error{db.dropFolder, db.dropMtimes, db.dropFolderMeta, db.folderIdx.Delete} {
    if err := f([]byte(folder)); ...
```

Reply:
I think that's a lot less readable than just having each call on its own line :)

Reply:
Actually you are right, I was just looking at the minus diff not the new code.